internal/pkg/constants/constants.go

```diff
@@ -44,4 +44,7 @@ const (
 
 	// Finalizers
 	KubewardenFinalizer = "kubewarden"
+
+	// Kubernetes
+	KubernetesRevisionAnnotation = "deployment.kubernetes.io/revision"
```

Comment:
You can use this one: https://github.com/kubernetes/kubernetes/blob/2da1558909043ddc41d7a36ab93672869474ec55/pkg/controller/deployment/util/deployment_util.go#L48

Reply:
You are right. I checked this by grepping on `kubernetes/staging`, that includes all repos that end up in separate repos in the Github `kubernetes/xyz` and there is not such `KubernetesRevisionAnnotation` constant we can use, only from the main tree, that should not be consumed directly.

Sorry for the bad pointer.